Jenkinsfile.windows

```diff
@@ -0,0 +1,281 @@
+
+                            // windows docker container cannot reach instance-metadata endpoint. supply credentials at upload.
+                            withAWS(role: 'jenkins', roleAccount: AWS_ACCOUNT_ID) {
+                              retry(5) {
+                                bat "aws s3 cp package\\win32\\build\\${packageName}.exe ${buildDest}/${packageName}.exe"
+                                bat "aws s3 cp package\\win32\\build\\${packageName}.zip ${buildDest}/${packageName}.zip"
+                              }
+                            }
+                          }
+                        }
+                      }
+
+                      stage('Sentry Upload') {
+                        environment {
+                          SENTRY_API_KEY = credentials('ide-sentry-api-key')
+                        }
+
+                        steps {
+                          retry(5) {
+                            // convert the PDB symbols to breakpad format (PDB not supported by Sentry)
+                            timeout(activity: true, time: 15) {
+                              bat '''
+                                cd package\\win32\\build
+                                FOR /F %%G IN ('dir /s /b *.pdb') DO (..\\..\\..\\dependencies\\windows\\breakpad-tools-windows\\dump_syms %%G > %%G.sym)
+                              '''
+
+                              // upload the breakpad symbols, attempt to run sentry uplaod
+                              bat "cd package\\win32\\build\\src\\cpp && ..\\..\\..\\..\\..\\dependencies\\windows\\sentry-cli.exe --auth-token %SENTRY_API_KEY% upload-dif --log-level=debug --org rstudio --project ide-backend -t breakpad ."
+                            }
+                          }
+                        }
+                      }
+
+                      stage ('Publish') {
+                        environment {
+                          GITHUB_LOGIN = credentials('github-rstudio-jenkins')
+                        }
+
+                        steps {
+                          script {
+                            def packageName = "RStudio-${PACKAGE_NAME}"
+
+                            // derive product
+                            def product = "${FLAVOR}"
+                              if (IS_PRO) {
+                                product = "${FLAVOR}-pro"
+                              }
+
+                            // publish the build (self installing exe)
+                            def stdout = powershell(returnStdout: true, script: ".\\docker\\jenkins\\publish-build.ps1 -build ${product}/windows -url https://s3.amazonaws.com/rstudio-ide-build/${current_container.flavor}/windows/${packageName}.exe -pat %GITHUB_LOGIN_PSW% -file package\\win32\\build\\${packageName}.exe -version ${rstudioVersionMajor}.${rstudioVersionMinor}.${rstudioVersionPatch}${rstudioVersionSuffix}")
```

Comment:
```suggestion
                            def stdout = powershell(returnStdout: true, script: '.\\docker\\jenkins\\publish-build.ps1 -build ' + product + '/windows -url https://s3.amazonaws.com/rstudio-ide-build/%FLAVOR%/windows/' + packageName + '.exe -pat %GITHUB_LOGIN_PSW% -file package\\win32\\build\\' + packageName + '.exe -version %RSTUDIO_VERSION%')
```

Reply:
I can change this, but it doesn't look like there is anything particularly sensitive being interpolated.